pyproject.toml

```
@@ -13,7 +13,7 @@ requires = [
 build = "cp310-* cp311-* cp312-* cp313-*"
 build-verbosity = 1
 test-requires = ["pytest", "pytest-cov", "PyYAML", "scipy"]
-before-test = "pip install --no-deps sparse@git+https://github.com/pydata/sparse@updated-llvm-nightly-test"
+before-test = "pip install --no-deps sparse@git+https://github.com/pydata/sparse@main"
```

Let's use the default branch.

```suggestion
before-test = "pip install --no-deps sparse@git+https://github.com/pydata/sparse.git"
```